src/colorizer/utils/url_utils.ts

```diff
@@ -24,10 +24,10 @@ const URL_PARAM_PALETTE = "palette";
 const URL_PARAM_PALETTE_KEY = "palette-key";
 
 const ALLEN_FILE_PREFIX = "/allen/";
-const ALLEN_PREFIX_TO_HTTPS: Record<string, string> = {
-  "/allen/aics/assay-dev": "https://dev-aics-dtp-001.int.allencell.org/assay-dev",
-  "/allen/aics/microscopy": "https://dev-aics-dtp-001.int.allencell.org/microscopy",
-};
+const ALLEN_PREFIX_TO_HTTPS: Map<string, string> = new Map([
```

Comment:
was this change required by the new lint rules?

Reply:
Yeah, it unfortunately disallows object properties that start in non-alphabetic or "_" characters. Record is really just a type constraint around a Javascript object...

Reply:
I see. Ok that makes sense!

Reply:
Not at all a blocking concern, but it does bug me a bit that this is compelling a (subtle, unimportant) behavior change in the code, since I believe `Map`s are represented differently and afford the interpreter fewer opportunities for optimization of constant properties. (That "I believe" makes this an extra unimportant criticism, since, like any of us, I have no way of knowing what V8 actually does with any of our code!)

Reply:
I agree - my sense is that it would be better to have a linter ignore comment than to be forced to use a different type that has potentially different behavior.  In this particular case I don't think it really matters, since this is such a simple scenario 😀